internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -144,6 +144,12 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 					} else {
 						endpoint = strings.Replace(endpoint, "/{account_or_zone}/{account_or_zone_id}/", "/zones/{zone_id}/", 1)
 					}
+				} else if strings.Contains(endpoint, "{accounts_or_zones}") {
```

Comment:
everything has swapped to this string now so we shouldn't need the if/else handling, right?